packages/backend/src/plugins/events.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EventsBuilder } from '@backstage/plugin-events-backend';
+import { HttpEndpointEventPublisher } from '@backstage/plugin-events-backend-module-http';
+import { AwsSqsEventPublisher } from '@backstage/plugin-events-backend-module-sqs';
+import { EventSubscriber } from '@backstage/plugin-events-node';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+  subscribers: EventSubscriber<any>[],
+): Promise<Router> {
+  const http = HttpEndpointEventPublisher.fromConfig({
+    config: env.config,
+    logger: env.logger,
+  });
+  const router = http.createRouter();
+
+  const sqs = AwsSqsEventPublisher.fromConfig({
```

Comment:
is this a bit much to add to the example backend? since it starts up an aws client and stuff

Reply:
without any config, it would return an empty array. Therefore, I think it is fine to add it there. However, we can drop it, too. The basic use case would be to use the http endpoints I guess.

Reply:
@freben let me know if you prefer to have it removed from the example backend.

Reply:
ok, then I will drop the sqs module from the backend example